graphql_api/types.py

```diff
@@ -784,3 +783,21 @@ class GatesIntegrationData(graphene.ObjectType):
     missing_tokenizer = graphene.Boolean()
     missing_embeddings = graphene.List(graphene.String)
     missing_information_sources = graphene.List(graphene.String)
+
+
+class AdminMessage(SQLAlchemyObjectType):
+    class Meta:
+        model = models.AdminMessage
+        interfaces = (Node,)
+
+    id = graphene.ID(source="id", required=True)
+
+
+class Task(graphene.ObjectType):
+    id = graphene.ID()
```

Comment:
should we maybe extend this by a start / end date so it's easier to detect "faulty" ones?

Reply:
just checked not all have start/end date so this would be more of an extension issue imo. (so adding start & end date for all types or something but not atm)

Reply:
Agree, would be really useful but most certainly in a later point of time